tests/test_other.py

```diff
@@ -8795,7 +8795,7 @@ def test_closure_full_js_library(self):
     # are including a lot of JS without corresponding compiled code for it. This still
     # lets us catch all other errors.
     with env_modify({'EMCC_CLOSURE_ARGS': '--jscomp_off undefinedVars'}):
-      run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.c'), '-O2', '--closure', '1', '-g1', '-s', 'INCLUDE_FULL_LIBRARY=1', '-s', 'ERROR_ON_UNDEFINED_SYMBOLS=0'])
+      run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.c'), '-O1', '--closure', '1', '-g1', '-s', 'INCLUDE_FULL_LIBRARY=1', '-s', 'ERROR_ON_UNDEFINED_SYMBOLS=0'])
```

Comment:
why did this change?

Reply:
-O2 performs dead code elimination that removes functions from the test, resulting in a test run that goes against `-s INCLUDE_FULL_LIBRARY=1`, and as result, Closure complains that some variables that are seen are never written to.